.polygon/rest.go

```diff
@@ -0,0 +1,46 @@
+package main
+
+import (
+	"bytes"
+	"encoding/json"
+	"io"
+	"log"
+	"net/http"
+	"net/url"
+	"os"
+)
+
+func main() {
+	if len(os.Args) < 2 {
+		log.Fatal("expected a URI for fetching the REST spec")
+	}
+	uri := os.Args[1]
+
+	var body []byte
+	if isURL(uri) {
+		resp, err := http.Get(uri)
+		if err != nil {
+			log.Fatal(err)
+		}
+		defer resp.Body.Close()
+
+		body, err = io.ReadAll(resp.Body)
+		if err != nil {
+			log.Fatal(err)
+		}
+	} else {
+		log.Fatal("expected arg to be a valid URI")
+	}
+
+	var out bytes.Buffer
+	err := json.Indent(&out, body, "", "  ")
```

Comment:
Can a Python script to do this instead? Go feels like a rather heavy-weight dependency to HTTP Get + format JSON.

Reply:
yeah, good point. i think it made sense in the go client but feels weird here. i'll refactor.

Reply:
Cool. I think something like this is close:
```
import urllib.request
import json

contents = urllib.request.urlopen("http://example.com/foo/bar").read()
parsed = json.loads(contents)
formatted = json.dumps(parsed, indent=2)
```

Reply:
done!